### PR TITLE
Add new tests

### PR DESCRIPTION
**Pull request by Avi-Eyal**
Add new automation tests for JWT authn:
9515
9516

### CODE REVIEW

File: cucumber/authenticators_jwt/features/authn_status_jwt.feature [hunk 1]
```diff
@@ -640,3 +640,131 @@ Feature: JWT Authenticator - Status Check
...
+
+    - !grant
+      role: !group conjur/authn-jwt/raw/operators
+      member:
+      - !user alice
+    """
+    And I am the super-user
+    And I successfully set authn-jwt jwks-uri variable with value of "myJWKs.json" endpoint
+    And I successfully set authn-jwt "token-app-property" variable to value "user"
+    And I successfully set authn-jwt "issuer" variable to value "gitlab"
```
Comment (tzheleznyak) [inline cucumber/authenticators_jwt/features/authn_status_jwt.feature:697]: I think that the issuer is irrelevant for this test

File: cucumber/authenticators_jwt/features/authn_status_jwt.feature [hunk 2]
```diff
@@ -640,3 +640,131 @@ Feature: JWT Authenticator - Status Check
...
+    And I login as "alice"
```
Comment (tzheleznyak) [inline cucumber/authenticators_jwt/features/authn_status_jwt.feature:698]: If we use jwks-uri we should use myapp host and not alice user?

File: cucumber/authenticators_jwt/features/authn_status_jwt.feature [hunk 3]
```diff
@@ -640,3 +640,131 @@ Feature: JWT Authenticator - Status Check
...
+    And I save my place in the log file
+    When I GET "/authn-jwt/raw/cucumber/status"
+    Then the HTTP response status code is 500
+    And the authenticator status check fails with error "CONJ00037E Missing value for resource: cucumber:variable:conjur/authn-jwt/raw/identity-path>"
+
+  Scenario: Valdi status check, identify-path is configured with value
```
Comment (tzheleznyak) [inline cucumber/authenticators_jwt/features/authn_status_jwt.feature:704]: Validi

File: cucumber/authenticators_jwt/features/authn_status_jwt.feature (hunk 2 above)
Comment (Avi-Eyal) [reply cucumber/authenticators_jwt/features/authn_status_jwt.feature:698]: this is the user thats making the status request, so as far as i know it should be a user and not a host

**Review by tzheleznyak — APPROVED** [review]
LGTM

**Pull request merged**